wire-library/wire-gradle-plugin/src/test/projects/custom-output-no-such-class/build.gradle

```diff
@@ -9,4 +9,7 @@ wire {
   custom {
     customHandlerClass = "NoSuchClass"
   }
+
+  java{
+  }
```

Comment:
Likewise, I don't think the JavaTarget was defined here

Reply:
Good catch will fix